portfolio/src/main/webapp/script.js

```diff
@@ -244,3 +244,37 @@ function closeWindows(map) {
     infoWindows[i].close(map, markers[i]);
   }
 }
+
+google.charts.load('current', {'packages':['corechart']});
+google.charts.setOnLoadCallback(drawChart);
+
+/** Creates a chart and adds it to the page. */
+function drawChart() {
+  const data = new google.visualization.DataTable();
+  data.addColumn('string', 'Task');
+  data.addColumn('number', 'Minutes');
+  data.addRows([
+    ['Deciding when to practice', 10],
+    ['Waiting to get picked up', 25],
+    ['Driving to/from where we\'re practicing', 20],
+    ['Waiting for everyone to show up', 30],
+    ['Socializing', 30],
+    ['Setting up gear', 15],
+    ['Playing songs that we aren\'t playing at our next gig', 20],
+    ['Deciding our set list', 30],
+    ['Rehearsing', 120],
+    ['Creating harmonies', 15],
+    ['Messing up an entrance or skipping a section', 25]
+  ]);
+
+  const options = {
+    'width': 650,
+    'height': 650,
+    'legend': 'none'
```

Comment:
Nit: Do you need to specify legend: 'none' ?

Reply:
Yes, because the labels weren't fully displaying on the legend and there didn't seem to be an easy fix to increase the legend with. The labels are still visible when you scroll over each slice.